paymentsheet/src/main/java/com/stripe/android/customersheet/analytics/CustomerSheetEvent.kt

```diff
@@ -206,6 +237,14 @@ internal sealed class CustomerSheetEvent : AnalyticsEvent {
         const val CS_UPDATE_PAYMENT_METHOD = "cs_update_card"
         const val CS_UPDATE_PAYMENT_METHOD_FAILED = "cs_update_card_failed"
 
+        const val FIELD_GOOGLE_PAY_ENABLED = "google_pay_enabled"
+        const val FIELD_BILLING = "default_billing_details"
+        const val FIELD_PREFERRED_NETWORKS = "preferred_networks"
+        const val FIELD_CUSTOMER_SHEET_CONFIGURATION = "cs_config"
```

Comment:
Will update `Zoolander` with this option prior to tracking the event inside of `CustomerSheetViewModel`